### ISSUE TITLE
Duplicate color ? 

### ISSUE THREAD

**Issue by LukyVj** [Diastro/github-colors#2]
Hi, it seems the `objective-c` color is mentionned twice, with 2 different color values. 
https://github.com/Diastro/github-colors/blob/master/github-colors.scss#L27
Line 4 & Line 27

**Comment by LukyVj** [comment]
And also the `c` is present three times.

**Comment by Diastro** [comment]
Yeah, one is Objective-C++ the other one is Objective-C but the contributer that added scss sanitized the key with Slugify... Same logic for c (C++, C#, C)
